setup.py

```diff
@@ -1,28 +1,26 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
-try:
-    with open("README.md") as f:
-        long_description = f.read()
-except OSError:
-    long_description = ""
+with open("README.md") as f:
+    long_description = f.read()
 
 setup(
     # Metadata
     name="JSON_minify",
     version="0.2.0",
+    include_package_data=True,
+    license='MIT License',
     description=(
-        "A simple script to minify valid JSON, containing C/C++ style comments"
+        "A simple script to minify valid JSON, containing JavaScript comments"
     ),
     long_description=long_description,
-    url="https://github.com/getify/JSON.minify/tree/python",
+    url="https://github.com/fiduswriter/JSON.minify/tree/python",
```

Comment:
You'll want to revert this line change before this merges. :P

Reply:
right. if you guys are willing to take over maintainership, we cna move it all back into the getify repository.